<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Crear</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>311</width>
     <height>95</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="nombre_3">
        <property name="text">
         <string>Nombre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nombre"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="cantJugadores_2">
        <property name="text">
         <string>cantidad de jugadores</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="cantJugadores"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="cantVueltas_2">
        <property name="text">
         <string>cantidad de vueltas</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="cantVueltas"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>151</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Pistas:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="pista1">
        <property name="text">
         <string>Pista 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pista2">
        <property name="text">
         <string>Pista 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pista3">
        <property name="text">
         <string>Pista 3</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
